salt/states/netacl.py

```diff
@@ -428,7 +427,11 @@ def term(name,
             - filter_options:
                 - not-interface-specific
             - term_name: {{ term_name }}
-            - {{ my_term_cfg }}
+            - {{ my_term_cfg | json }}
+
+    When passing retrieved pillar data into the state file, it is strongly
+    recommended to use the json serializer explicitly, instead of relying
```

Comment:
Perhaps it would be useful to add something like "see example below" or even have something like:

> recommended to use the json serializer explicitly (`| json`)

What do you think? From your perspective how would you find more intuitive?